packages/clay-css/src/scss/variables/_alerts.scss

```diff
@@ -177,11 +177,13 @@ $alert-link: map-deep-merge(
 // .alert-indicator
 
 $alert-indicator-font-size: 1.25rem !default;
+$alert-indicator-line-height: 1 !default;
```

Comment:
We don't need to add this specific variable here, just add `line-height: 1,` in the `$alert-indicator` variable.